src/base/bittorrent/session.cpp

```diff
@@ -2208,6 +2208,7 @@ bool Session::addTorrent_impl(CreateTorrentParams params, const MagnetUri &magne
         if (params.restored) {  // load from existing fastresume
             lt::error_code ec;
             p = lt::read_resume_data(fastresumeData, ec);
+            p.save_path = Profile::instance().fromPortablePath(Utils::Fs::toUniformPath(fromLTString(p.save_path))).toStdString();
```

Comment:
Converting `save_path` from relative to absolute after reading fast resume data.

Reply:
IIRC, builds that linked with libtorrent-1.1.x aren't affected by this issue, isn't it?

Reply:
I'm sorry but I didn't try to link with libtorrent-1.1.x, only with latest libtorrent-1.2.3. Should I also check if the fix is needed for libtorrent-1.1.x ? Is possibility of linking with libtorrent-1.1.x left for backwards compatibility or maybe it's working differently (better?) in some cases?

Reply:
Added patching of `save_path` for old libtorrent.